pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
```

Comment:
Hey @sbrl - its about the natural flow of actions that a user will do. Usually the user will first use composer to install and remove package dependencies, and then update `composer` when needed. 

We try to mention the options in the order of the natural flow of usage.